MapboxNavigation/NavigationViewController.swift

```diff
@@ -259,7 +285,7 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         
         super.init(contentViewController: mapViewController, drawerViewController: tableViewController)
         
-        self.styles = styles ?? [DefaultStyle()]
+        self.styles = styles ?? [DefaultStyle(), DefaultDarkStyle()]
```

Comment:
I'm sort of confused as to why we have a default value but also need this `??` here.